device/nokia/x86_64-nokia_ixr7250e_sup-r0/Nokia-IXR7250E-SUP-10/0/config-ramon-1-0.bcm

```diff
@@ -2,7 +2,7 @@
 appl_enable_intr_init.BCM8879X=0
 appl_param_module_id.BCM8879X=1
 bcm_linkscan_interval.BCM8879X=0
-bcm_stat_interval.BCM8879X=4000000
+bcm_stat_interval.BCM8879X=0
```

Comment:
Making this to zero, will this affect the fabric counter poll ?

Reply:
No. It will pull the counters from HW on demand